react-app/src/components/Forms/UncontrolledForm.tsx

```diff
@@ -0,0 +1,339 @@
+import { object, string, number, mixed, bool, ValidationError } from 'yup';
```

Comment:
please create other dedicated modules for validator and other staff. 300 lines is too much for a single component